tests/test_logging.py

```diff
@@ -1,7 +1,57 @@
 import pytest
 
-from mlserver.logging import logger, configure_logger
-from mlserver.settings import Settings
+from mlserver import ModelSettings
+from mlserver.context import model_context
+from mlserver.logging import (
+    ModelLoggerFormatter,
+    LoggerFormat,
+    configure_logger,
+    logger,
+)
+from mlserver.settings import ModelParameters, Settings
+from tests.fixtures import SumModel
+from logging import INFO
+
+
+@pytest.mark.parametrize(
+    "name, version, expected_fmt",
```

Comment:
:wrench: The implementation of the model logger allows the model name to be empty.  This case should either be captured by a test case or it shouldn't be permitted because it indicates something has gone wrong (or hasn't been wired up correctly).